src/igvfd/schemas/analysis_step.json

```diff
@@ -0,0 +1,130 @@
+{
+    "title": "Analysis Step",
+    "description": "Schema for submitting a step in a computational analysis workflow.",
+    "$id": "/profiles/analysis_step.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "title",
+        "step_label",
+        "analysis_step_types",
+        "input_content_types",
```

Comment:
should not output_types be required as well?

Reply:
That makes sense, will update